src/net/udp.rs

```diff
@@ -175,6 +175,12 @@ impl UdpSocket {
         self.inner.send_to(buf, socket_addr).await
     }
 
+    /// Sends data on the socket. Will attempt to do so without intermediate copies.
+    /// On success, returns the number of bytes written.
```

Comment:
I haven't read about the zero copy operation added to the uring interface. Is there any reason an app would still prefer to use send rather than send_zc? Maybe we could give some guidance here when one might use one over the other.

Reply:
So something would be worth saying, either about the tradeoffs as you described them in this review, or how one might reason about using one or the other.